proto/tendermint/abci/types.proto

```diff
@@ -262,6 +301,34 @@ message ResponseApplySnapshotChunk {
   }
 }
 
+message ResponsePrepareProposal {
+  bool                             modified_tx             = 1;
+  repeated TransactionRecord       txs                     = 2;
```

Comment:
nit: we use Tx in most places to mean Transaction, I think it's worth keeping that pattern up here and also naming the field `tx_records`.

Reply:
Totally agree! Changed. Please double check